db/seeds.rb

```diff
@@ -47,3 +49,79 @@ class SeedAdminENVError < StandardError
 business_categories.each { |b_category| BusinessCategory.find_or_create_by(name: b_category) }
 BusinessCategory.find_or_create_by(name: 'Sociala tjänstehundar', description: 'Terapi-, vård- & skolhund dvs hundar som jobbar tillsammans med sin förare/ägare inom vård, skola och omsorg.')
 BusinessCategory.find_or_create_by(name: 'Civila tjänstehundar', description: 'Assistanshundar dvs hundar som jobbar åt sin ägare som service-, signal, diabetes, PH-hund mm')
+
+if Rails.env.development? || Rails.env.staging?
+  r = Random.new
+  USERS = 40
+  regions = Region.all.to_a # regions are created in a migration file
+  num_regions = regions.size
+
+  # Create users
+  users = []
+
+  USERS.times do
+    users << User.create(email: FFaker::InternetSE.free_email,
```

Comment:
"whatever" is the default seeded password. (see [line 44](https://github.com/patmbolger/shf-project/blob/a507c53740a005bb45c22235241da75cbd1b6fc2/db/seeds.rb#L44)

Would probably now be worth pulling that out into a constant.